src/screens/GroupInfo/index.js

```diff
@@ -449,5 +453,18 @@ export const styles = StyleSheet.create({
   },
   scrollContainer: {
     paddingBottom: 30
+  },
+  modalActUser: {
+    padding: 16,
+    paddingLeft: 27,
+    paddingRight: 70
+  },
+  textActUser: {
+    fontFamily: 'Inter',
+    fontStyle: 'normal',
+    fontWeight: '500',
+    fontSize: 18,
+    color: COLORS.black000,
+    paddingVertical: 20
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new styles added to the `StyleSheet.create` function seem fine. However, ensure that these styles (`modalActUser` and `textActUser`) are used in the appropriate places in your components. Also, remember to maintain consistency in your codebase by following the same style naming conventions used elsewhere.
```diff
  scrollContainer: {
    paddingBottom: 30
  },
+ modalActUser: {
+   padding: 16,
+   paddingLeft: 27,
+   paddingRight: 70
+ },
+ textActUser: {
+   fontFamily: 'Inter',
+   fontStyle: 'normal',
+   fontWeight: '500',
+   fontSize: 18,
+   color: COLORS.black000,
+   paddingVertical: 20
+ }
});
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->